src/main/java/Incy.java

```diff
@@ -0,0 +1,77 @@
+                System.out.println("____________________________________________________________");
+                if (taskCount == 0) {
+                    System.out.println("Blimey, your list is empty, innit?");
+                } else {
+                    for (int i = 0; i < taskCount; i++) {
+                        System.out.println((i + 1) + ". " + tasks[i]);
+                    }
+                }
+                System.out.println("____________________________________________________________");
+            } else if (input.startsWith("mark ")) {
+                int index = Integer.parseInt(input.substring(5)) - 1;
+                if (index >= 0 && index < taskCount) {
+                    tasks[index].markAsDone();
+                    System.out.println("____________________________________________________________\n"
+                                        + "Buzzin'! This one's sorted now:\n  "
+                                        + tasks[index]
+                                        + "\n____________________________________________________________"
+                    );
+                }
+            } else if (input.startsWith("unmark ")) {
+                int index = Integer.parseInt(input.substring(7)) - 1;
+                if (index >= 0 && index < taskCount) {
+                    tasks[index].markAsNotDone();
+                    System.out.println("____________________________________________________________\n"
+                                        + "No prob, flipped it back to not done, innit:\n  "
+                                        + tasks[index]
+                                        + "\n____________________________________________________________"
+                    );
+                }
+            } else {
+                if (taskCount >= tasks.length) {
+                    System.out.println("____________________________________________________________\n"
+                                        + "Cor blimey! The list is full to the brim. Can't add more, sorry!"
+                                        + "\n____________________________________________________________"
+                    );
+                } else {
+                    tasks[taskCount] = new Task(input);
+                    taskCount++;
+                    System.out.println("____________________________________________________________\n"
+                                        + "Sorted! Added: " + input
+                                        + "\n____________________________________________________________"
+                    );
+                }
+            }
+        }
+
+        System.out.println("____________________________________________________________\n"
+                            + "Cya later mate!"
+                            + "\n____________________________________________________________"
+        );
+        scanner.close();
+    }
```

Comment:
This code block is deeply nested, and needs to be worked on. 

In general, this method is too long, and it is preferred for methods to be shorter than 30 lines of code. 

Please work on abstracting this chunk of code.

References: 
1. https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-deep-nesting
2. https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods